src/components/molecules/Common/Header/index.tsx

```diff
@@ -107,13 +107,17 @@ const Wrapper = styled.header`
   width: 100%;
   height: ${metrics.headerHeight}px;
   background: ${props => props.theme.header.bg};
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
Little confused why this is necessary. The Content component declared below has flex. Also this Wrapper only has Content as its child so flex and flex-direction as far as I can tell would have no affect. Let me know if I am missing something.